tox.ini

```diff
@@ -15,8 +15,11 @@ deps =
     pytest
     pytest-cov
     pytest-icdiff
+setenv =
+    # Display up to 20 frames in backtraces when showing ResourceWarnings.
+    PYTHONTRACEMALLOC=20
 commands =
-    pytest --strict-markers --cov {posargs}
+    python -Wall -m pytest --strict-markers --cov {posargs}
```

Comment:
Curious why you went with the outer call instead of using `pytest -Wall ...`?
See https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html

Reply:
I think this is addressed now.